components/Link/Link.Styled.js

```diff
@@ -15,7 +15,7 @@ export const StyledLink = styled(Link)`
     rgba(0, 102, 51, 1) 58%
   );
   font-size: 1rem;
-  padding: 0.5rem 0.5rem;
+  padding: 0.7rem 0.7rem;
```

Comment:
The super-duper short hand version:

```suggestion
  padding: 0.7rem;
```